app/sidekiq/central_mail/submit_central_form686c_job.rb

```diff
@@ -136,6 +156,17 @@ def generate_metadata
       metadata.merge(generate_attachment_metadata(attachment_paths))
     end
 
+    def generate_metadata_lh
+      {
+        veteranFirstName: form['veteran_information']['full_name']['first'],
+        veteranLastName: form['veteran_information']['full_name']['last'],
+        fileNumber: form['veteran_information']['file_number'] || form['veteran_information']['ssn'],
+        zipCode: address['country_name'] == 'USA' ? address['zip_code'] : FOREIGN_POSTALCODE,
+        docType: claim.form_id,
+        claimDate: claim.created_at
```

Comment:
Check with the CMP intake people if they actual respect this claimDate param from sources other than the 526 backup process. IIRC they added specific logic that explicitly only supported this metadata for the 526 process. They will want to extend that support to this process as well obviously.

Reply:
Just because you are sending in both it does not mean they are respecting it. You can send any data in that meta-data, if they do anything with it is up to them though. So doesn't hurt to doublecheck with them!